doc/classes/TileMap.xml

```diff
@@ -76,11 +76,10 @@
 				Clears cells that do not exist in the tileset.
 			</description>
 		</method>
-		<method name="force_update" is_deprecated="true">
+		<method name="force_update" deprecated="See [method notify_runtime_tile_data_update] and [method update_internals].">
 			<return type="void" />
 			<param index="0" name="layer" type="int" default="-1" />
 			<description>
-				[i]Deprecated.[/i] See [method notify_runtime_tile_data_update] and [method update_internals].
```

Comment:
I'm a bit peeved off by these kind of descriptions.

Obviously, `force_update` still works, so it needs to have a _somewhat_ meaningful description still, albeit extremely succinct, so that users are even more encouraged not to use it. Yet they would still know what it does if they encounter it in old code.

Deprecating without the "what" and "why" is is a bit malicious in my opinion.